### DISCUSSION TITLE
Local DNS not working if there is no internet connection

### DISCUSSION THREAD

**Discussion by BrunoLindorfer** [AdguardTeam/AdGuardHome#4778] [Q&A]
I have configured AdGuard Home to use my OpenWRT DHCP server for local domains ("lan") and an external DNS for all other requests
```
[/lan/]192.168.0.2
8.8.8.8
```
Yesterday there was a problem at my ISP and my internet connection failed. So the external DNS was not reachable. But as long as there was no internet also the local requests could not be resolved (the local DNS 192.168.0.2 was still reachable of course). Is there any setting to keep the local DNS working when there is no internet?
Thanks!

**Comment by ainar-g** [comment]
Good day.  That shouldn't really happen.  Which upstream mode are you using?  Does using parallel requests improve the situation?
